src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntity.kt

```diff
@@ -510,6 +510,7 @@ class TemporaryAccommodationApplicationEntity(
   var pdu: String?,
   var name: String?,
   var prisonReleaseTypes: String?,
+  var probationDeliveryUnitId: UUID?,
```

Comment:
I would expect this to be defined as a ManyToOne relationship to the ProbationDeliveryUnitEntity instead of a UUID. Is there a reason we haven't modelled it this way? As long as the ManyToOne annotation marks the relationship as fetch = lazy, there should be any performance penalty adding this (we've learnt that the hard way with spring boot 3 upgrade 😄 ) . The column name can remain as is.

Reply:
Super, thanks for this.  Should be ok now?

Reply:
Looks good!